src/main/java/com/example/petbuddybackend/entity/care/Care.java

```diff
@@ -0,0 +1,65 @@
+package com.example.petbuddybackend.entity.care;
+
+import com.example.petbuddybackend.entity.animal.Animal;
+import com.example.petbuddybackend.entity.animal.AnimalAttribute;
+import com.example.petbuddybackend.entity.user.Caretaker;
+import com.example.petbuddybackend.entity.user.Client;
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.Check;
+
+import java.time.LocalDate;
+import java.time.ZonedDateTime;
+import java.util.Set;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity
+@Getter @Setter
+@Table(
+        uniqueConstraints = { @UniqueConstraint(columnNames = { "caretakerEmail", "clientEmail", "careStart", "careEnd" }) }
+)
+@Check(constraints = "client_email <> caretaker_email")
+public class Care {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private ZonedDateTime submittedAt;
```

Comment:
Rozumiem że to data utworzenia opieki? To może by dodać coś w stylu

```java
    @PrePersist
    public void prePersist() {
        submittedAt = ZonedDateTime.now();
    }
```

Reply:
Też zapomniałem:)

Reply:
A to działa tak że jeśli ustawie ręcznie to zostanie to co ustawiłem ręcznie, czy nadpisze?

Reply:
Tak, przed zapisaniem encji w bazie to się wywoła